internal/worker/remoterelations/remoteapplicationworker.go

```diff
@@ -72,9 +82,9 @@ type relation struct {
 	localDead      bool
 	suspended      bool
 	localUnitCount int
-	localRuw       *relationUnitsWorker
-	remoteRuw      *relationUnitsWorker
-	remoteRrw      *remoteRelationsWorker
+	localRuw       ReportableWorker
```

Comment:
Change these so they are not so brutally abbreviated. Give some clue now that they are typed as indirections.